src/cloud-api-adaptor/docs/SecureComms.md

```diff
@@ -0,0 +1,98 @@
+# Secure Communications
+
+Here we describe how to setup the Secure Communications (`Secure Comms` for short) feature of the Adaptor and Forwarder
+
+Secure Comms feature establish an ssh channel by the Adaptor as an ssh client to the Forwarder as an ssh server. The ssh channel is than used for all control communication between the Adaptor and Forwarder. Secure Comms use a two phase approach:
+- Attestation Phase: During this phase an ssh channel is created to enable the Peer Pod to attest against an attestation service. The attestation traffic is tunneled from the Peer Pod to the CAA and from there sent to a KBS as configured at the CAA. Once attestation is achieved, the Forwarder obtains Peer Pod keys from the KBS. The Peer Pod keys are dynamically added to the KBS by the CAA prior to starting the Peer Pod. 
+- Kubernetes Phase: During this phase an ssh channel is created to enable Runtime communicate with teh Kata Agent. This phase uses the keys obtained during the Attestation Phase ensuring that the communication is secured. 
+
+Additional tunnels can be establish via the ssh channel by configuring the Adaptor and Forwarder as described below.
+
+## Setup
+
+### Deploy CAA
+Use any of the option for installing CAA depending on the cloud driver used.
+
+### Deploy KBS-Operator
+Deploy KBS-Operator by following instructions at [https://github.com/confidential-containers/kbs-operator?tab=readme-ov-file#getting-started](KBS Operator Getting Started). 
+
+Copy the kbs-client secret from the `kbs-operator-system` namespace to the `confidential-containers-system` ns. This can be done using:
+
+```sh
+kubectl get secret kbs-client -n kbs-operator-system -o json|jq --arg ns "confidential-containers-system" 'del(.metadata["creationTimestamp","resourceVersion","selfLink","uid","annotations"]) | .metadata.namespace |= $ns' |kubectl apply -f -
+```
+
+For a testing environment, you may need to change the policy of the KBS and AS using the KBS Client to allow all or fit your own policy. One way to do that is:
+
+```sh
+kubectl -n kbs-operator-system exec deployment/kbs-deployment --container as -it -- /bin/bash
+        apt update
+        apt install vim
+        vim /opt/confidential-containers/attestation-service/opa/default.rego // replace default policy to true
+
+kubectl -n kbs-operator-system exec deployment/kbs-deployment --container kbs -it -- /bin/bash
+        apt update
+        apt install vim
+        vim /opa/confidential-containers/kbs/policy.rego // replace default policy to true
+```
+
+
+
+### Build podvm to enforce Secure-Comms
```

Comment:
If we need to keep the re-built of the podvm to enable this then it might be worth linking to ./../podvm/README.md that has the instructions. We might also want to put this before the user has set up their Cloud api adaptor set-up and/or explain how to update the podvm that is being used?